otx/mpa/cls/stage.py

```diff
@@ -80,6 +80,21 @@ def is_mmov_model(k, v):
             cfg.merge_from_dict(data_cfg)
         self.configure_data(cfg, training, **kwargs)
 
+        if training:
+            if "unlabeled" in cfg.data and cfg.train_type == "SEMISUPERVISED":
+                update_or_add_custom_hook(
+                    cfg,
+                    ConfigDict(
+                        type="UnlabeledDataHook",
+                        unlabeled_data_cfg=cfg.data.unlabeled,
+                        samples_per_gpu=cfg.data.unlabeled.pop("samples_per_gpu", cfg.data.samples_per_gpu),
```

Comment:
Is this batch size for `labeled sample`?

Reply:
If we don't have any input for unlabeled batch size, batch size of labeled sample will be used.

Reply:
How to check? or Where to check if unlabeled_batch_size have or not?

Reply:
Check here with `pop`